moveit_core/robot_trajectory/src/robot_trajectory.cpp

```diff
@@ -274,8 +274,38 @@ void robot_trajectory::RobotTrajectory::getRobotTrajectoryMsg(moveit_msgs::Robot
     {
       trajectory.multi_dof_joint_trajectory.points[i].transforms.resize(mdof.size());
       for (std::size_t j = 0; j < mdof.size(); ++j)
+      {
         tf::transformEigenToMsg(waypoints_[i]->getJointTransform(mdof[j]),
                                 trajectory.multi_dof_joint_trajectory.points[i].transforms[j]);
+        if (waypoints_[i]->hasVelocities())
+        {
+          const std::vector<std::string> names = mdof[j]->getVariableNames();
+          const double* velocities = waypoints_[i]->getJointVelocities(mdof[j]);
+
+          geometry_msgs::Twist point_velocity;
+
+          for (std::size_t ii = 0; ii < names.size(); ++ii)
```

Comment:
instead of having an ``i``, ``j``, and ``ii`` iterator var, how about naming it ``k`` or something more descriptive?